resources/views/components/timeline.blade.php

```diff
@@ -10,13 +10,22 @@
     </div>
 
     <div class="flex flex-col gap-14 w-full">
-        @forelse ($incidents as $date => $incident)
-        <x-cachet::incident :date="$date" :incidents="$incident" :schedules="$schedules" />
-        @empty
-        <div class="text-zinc-500 dark:text-zinc-400 text-center">
-            {{ __('No incidents reported between :from and :to.', ['from' => $from, 'to' => $to]) }}
+            @foreach ($incidents as $date => $incident)
```

Comment:
Looking at this, I think that if incidents are empty, but schedules aren't, then nothing will show.